css/style.css

```diff
@@ -0,0 +1,104 @@
+img{
+ height: 500px;
+ width: 1300px;
+
+}
+
+h1{
+
+    position: absolute;
+    top:20px;
+   color: white ;
+   padding: 50px;
+}
+h2{
+    position: absolute;
+    top:20px;
+    left: 140px;
+    color:white;
+    padding: 150px;
+    font-size: 24px;
+    
+}
+
+h4{
+    position: absolute;
+    top:1000px;
+    left: 700px;
+    color:gray; 
+    text-align: center;
+    
+
+}
+
+.button1{
+    position: absolute;
+    top:300px;
+    left: 500px;
+    background-color:yellow;
+    font-size: 40px;
+    border: 5px border radius;
+    padding-left:20px ;
+
+
+}
+.Benefits{
+
+    background-color: #f0efef;
+    height: 300px;
+    width: 1300px;
+    padding: 10px;
+    font-size: 24px;
+    color: gray;
+
+
+}
+div img{
+    
+height: 300px;
+ width: 1300px;
+}
+
+.Relaxr1{
+    position: absolute;
+    top:900px;
+    left: 300px;
+    color:gray; 
+    text-align: center;
+}
+
+.signUp{
+
+    background-color: yellow;
+    height:100px;
+    width: 1300px;
+    margin: 0px 0px 0px 0px;
```

Comment:
```css
    margin: 0px;
```
if all sides of your margin/padding are the same size you can just type it once.